### PR TITLE
Implement adapter Name and add Name constant

### PR DESCRIPTION
**Pull request by lafriks**
~~Depends on go-rel/rel#346 and go-rel/sql#59~~

Add also `MustOpen` method similar to other adapters

### CODE REVIEW

File: sqlite3.go [hunk 1]
```diff
@@ -23,6 +23,18 @@ import (
 	"github.com/go-rel/sql/builder"
 )
 
+type sqlAdapter struct {
+	*sql.SQL
+}
+
+// Name of database type this adapter implements.
+const Name string = "sqlite3"
```
Comment (Fs02) [inline sqlite3.go:31]: I think we should stick with Name or DBType only?

Comment (lafriks) [reply sqlite3.go:31]: Do you mean to rename this same as func name?

Comment (Fs02) [reply sqlite3.go:31]: yes, or the otherway around is fine too
since we want to use this constant for comparison too, so better to be consistent

**Review by Fs02 — APPROVED** [review]
LGTM

**Pull request merged**